README.md

```diff
@@ -165,133 +158,105 @@ clicking the right button of your mouse on "Download Aspera Connect <version num
 "Copy link address".
 
 #### run it
+
 ```
 chmod +x ibm-aspera-connect_<version number>-linux_x86_64.sh
 
 ./ibm-aspera-connect_<version number>-linux_x86_64.sh
 ```
 
 #### add it to the system path
+
 ```
 export PATH=$PATH:~/.aspera/connect/bin/
 
 echo 'export PATH=$PATH:~/.aspera/connect/bin/' >> ~/.bash_profile
 ```
 
 #### check that everything works:
-`ascp --version`
 
+`ascp --version`
 
 Refer to the following sections for usage examples:
 
- - [Download data for a single SRP via Aspera CLI](#download-data-for-a-single-srp-via-aspera-cli)
- - [Download data for a single SRR via Aspera CLI](#download-data-for-a-single-srr-via-aspera-cli)
- 
+- [Download data for a single SRP via Aspera CLI](#download-data-for-a-single-srp-via-aspera-cli)
+- [Download data for a single SRR via Aspera CLI](#download-data-for-a-single-srr-via-aspera-cli)
+
 ### FTP
 
 FastqHeat will download files directly from ENA.
 
 Refer to the following sections for usage examples:
 
- - [Download data for a single SRP via FTP](#download-data-for-a-single-srp-via-ftp)
- - [Download data for a single SRR via FTP](#download-data-for-a-single-srr-via-ftp)
- 
-## Using FastqHeat
-
-For every study or run given, FastqHeat will download data for all runs and place them in
-a specific hierarchical directory structure.
-
-For example, if you wish to download data for `SRP163674` to `/some/output/directory`,
-FastqHeat will arrange downloaded files for runs in the following directory structure:
-
-```
-/some/output/directory/
```

Comment:
what's wrong with that? I guess it is good illustration that should stay in the readme

Reply:
I believe it is not the way it actually works.

Reply:
@dmitriy-grossman-quantori  It actually works like that, I've tried it while implementing other features